change/@ni-nimble-components-6e819e6f-2633-43f7-8e29-c38c7850ee59.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@fredvisser dang-it, I messed up in https://github.com/ni/nimble/pull/1836

bodyPlus1 is the `element` and Emphasized is the `state` for [token naming](https://github.com/ni/nimble/blob/33f4878a8ae07e14e0241693836a088a00ed7c23/packages/nimble-components/CONTRIBUTING.md#token-naming).

Can we update the body emphasized tokens to:
bodyPlus1Emphasized and
bodyPlus1

Reply:
Mentioned the change [on the issue](https://github.com/ni/nimble/issues/1831#issuecomment-2008278376) since that seems to be the [only user so far](https://dev.azure.com/ni/_search?action=contents&text=body-emphasized-plus-1&type=code&lp=custom-Collection&filters=&pageSize=25&result=DefaultCollection%2FDevCentral%2FTestStand%2FGBmain%2F%2Fsrc%2Fdev%2FArmstrong%2FSrc%2FControls%2FComponents%2FInternal%2FStepTable.razor.css)